internal/fnruntime/container.go

```diff
@@ -158,20 +161,29 @@ func NewContainerEnvFromStringSlice(envStr []string) *runtimeutil.ContainerEnv {
 // exist.
 func (f *ContainerFn) prepareImage() error {
 	// check image existence
-	args := []string{"image", "ls", f.Image}
+	foundImageInLocalCache := false
+	args := []string{"image", "inspect", f.Image}
 	cmd := exec.Command(dockerBin, args...)
 	var output []byte
 	var err error
-	if output, err = cmd.CombinedOutput(); err != nil {
-		return &ContainerImageError{
-			Image:  f.Image,
-			Output: string(output),
-		}
-	}
-	if strings.Contains(string(output), strings.Split(f.Image, ":")[0]) {
+	if _, err = cmd.CombinedOutput(); err == nil {
 		// image exists locally
+		foundImageInLocalCache = true
+	}
+
+	// If env var ImagePullPolicyEnv is set to "ifnotpresent", we scan the local
```

Comment:
Where are these envs documented? Should we mention them in the command help?

Reply:
I guess "Developing Functions" section in the Kpt book.